inline_scan.sh

```diff
@@ -537,9 +537,12 @@ save_and_copy_images() {
     save_file_name="${base_image_name}.tar"
     local save_file_path="${TMP_PATH}/${save_file_name}"
 
-    # Eventually remove localbuild from FULLTAG is present or docker save crashes
-    local save_img=$(echo ${FULLTAG} | sed "s/localbuild\///g")
-    docker save "${save_img}" -o "${save_file_path}"
+    local image_name=$(echo ${SCAN_IMAGES[0]} | rev | cut -d '/' -f 1 | rev )
```

Comment:
Is there any case where we need the additional modifications that `FULLTAG` has? It does indeed make sense to use the user supplied image name in case it contains a colon or the same with `:latest` if it doesn't; just want to make sure that there are no other corner cases we can think of.

Reply:
FULLTAG prefix with `localbuild` or `docker.io`, and eventually suffix with `:latest`.

Afaik, we are covering all we need. Also, in this way the behaviour is basically the same of a `docker pull <image>`